src/concepts/components/styling.md

```diff
@@ -0,0 +1,36 @@
+# Styling
+
+&lt;Work in progress&gt;
+
+Proposal for proper CSS support can be found here: [https://github.com/yewstack/yew/issues/533](https://github.com/yewstack/yew/issues/533)
+
+
+In the meantime you can define the style with any standard CSS tools
+
+## Yew component does not support yet class attribute
+
+So far Yew component does not support natively the `class` attribute. You can add the class attribute to your component:
```

Comment:
Gramatically this should read more along the lines of "At the moment Yew's components don't natively support the `class` attribute."

Reply:
I understand that this limitation is now resolved with the 0.16 release with #1187
=> I remove the chapter